cypress/integration/constraint-creation.feature

```diff
@@ -37,3 +37,8 @@ Feature: create constraints by selecting nodes from the tree
     When I select patients that are part of study Oracle_1000_Patient with age between 50 - 55  and numerical_1 between 0 - 10
     Then there are '495' subjects
 
+  Scenario: create a simple sample based subselection constraint
```

Comment:
a little confused with the terminology, does 'sample-based' only refer to diagnosis? or it meant for all of the four: subject, diagnosis, biomaterial and bio-source?